### DISCUSSION TITLE
Deployment successful but .ts files not being found

### DISCUSSION THREAD

**Discussion by steelballs** [vercel/community#1227] [Help] [answered]
### Summary

So I performed a deployment on Friday at 13:53 and everything worked as expected. About 2 hours later I did a very small update, literally added 1 line to a JSON definition and deployed once again. Started seeing errors with the .ts file now not being reachable the server was trying to find .js versions.

The only difference I could spot between the 2 deployments, other than the very simple update, was that the Vercel CLI version used between the deployments appears to have been upgraded to a new version.

<img width="926" alt="Screenshot 2023-01-07 at 17 54 21" src="https://user-images.githubusercontent.com/6048892/211203233-ff7a3211-1d85-449c-bf67-6acd0e6a4394.png">
<img width="927" alt="Screenshot 2023-01-07 at 17 55 09" src="https://user-images.githubusercontent.com/6048892/211203235-4707cfc3-7e3e-4c91-9176-dc65b838e48d.png">

All subsequent efforts to fix the issues have failed.

Can you let me know what changed between Vercel CLI 28.10.2 and Vercel CLI 28.10.3? 

Thanks

**Comment by swarnava** [comment]
Hi,

You can see change logs here: https://github.com/vercel/vercel/releases/

Can you set following environment variable temporarily and try to rebuild?

`VERCEL_CLI_VERSION=vercel@28.10.2`

**Comment by styfle** [comment]
If you have tsconfig.json with `module: esnext`, then your package.json should have `type: module`.

If you don’t want to use ESM, you can change tsconfig.json to `module: commonjs`.

**Comment by amyegan** [comment, marked as answer]
Update from the related thread: https://github.com/vercel/community/discussions/1225#discussioncomment-4642381
